cron/releasetest/k8s/controller.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: scorecard-batch-controller-releasetest
+spec:
+  # Daily at midnight PST.
+  schedule: "0 7 * * *"
+  concurrencyPolicy: "Forbid"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: controller
+            image: gcr.io/openssf/scorecard-batch-controller-releasetest:latest
```

Comment:
is it possible to pin this? Does dependabot support updating it?
I have a tracking issue to add support for this in our Frozen-Deps check.

Reply:
Hmm, the idea is that this should always pull the most recent image to test. Pinning doesn't really make sense here.